core/domain/change_domain.py

```diff
@@ -223,7 +231,7 @@ def to_dict(self):
 
         return base_change_dict
 
-    def __getattr__(self, name: str) -> str:
+    def __getattr__(self, name: str) -> Any:
```

Comment:
Why `Any`?

Reply:
Mypy suggested that changes
![image](https://user-images.githubusercontent.com/65958420/148024654-acd1d2f3-e4bf-447c-9557-738103b0874c.png)
and it does not depend on any previous annotation.

Reply:
I think `assert` should be used here.

Reply:
Thanks, it worked!

Reply:
Missing explanatory comment for this `Any`.